syntax = "proto3";

import "igdbapi.proto";
import "steam_entry.proto";

package espy;

message ReconciliationTaskList {
  repeated ReconciliationTask task = 1;
}

message ReconciliationTask {
  SteamEntry steam_entry = 1;
  repeated ReconciliationCandidate candidate = 2;
}

message ReconciliationCandidate {
  igdb.Game game = 1;
  int32 score = 2;
}
